src/components/CityWeatherCard.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styles from './CityWeatherCard.module.css';
+
+export interface CityWeatherCardProps {
+    name: string;
+    temperature: string;
+    conditions: string;
+    wind_speed: string;
+    humidity: string;
+}
+
+const CityWeatherCard: React.FC<CityWeatherCardProps> = ({
+    name,
+    temperature,
+    conditions,
+    wind_speed,
+    humidity
+}) => {
+    return (
```

Comment:
NIT: Looks like this can be shortened to remove the return statement since the arrow function will return anything to the right hand side of the arrow.

E.G
```tsx
// instead of this
const CityWeatherCard = (...) => {
  return (
    <div className={styles.card}>
      ...
      
     </div>
   );
 };

// the same can be achieved with
const CityWeatherCard = (...) => (
  <div className={styles.card}>
      ...
      
   </div>
 );
 
 // this also works for other types, eg:
 const add = (a: number, b: number): number => a + b;
 const concat = (a: string, b: string): string => `${a}${b}`;
 ...
 ```

Reply:
nice tip Will didn't link the two, neatens up things so I like it, cheers for explanation aswell